### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by AmaanSayyad**
### README.md changes list

1. Grammatical Mistakes Encountered.
2. Text is simplified.
3. Summary Enhancement to make repo more readable.

### CODE REVIEW

**Review by jacobcreech — CHANGES_REQUESTED** [review]

**Comment by jacobcreech** [comment]
Thanks for the smaller update. Just as a heads up, this PR would not be able to get the NFT if you're interested. It is only for code contributions.

**Comment by Agunderman10** [comment]
> Thanks for the smaller update. Just as a heads up, this PR would not be able to get the NFT if you're interested. It is only for code contributions.

This should probably be added to the README for new contributors to know they need to make code changes

File: README.md [hunk 1]
```diff
@@ -84,6 +84,6 @@ You can find tasks on the [project board](https://github.com/solana-dev-adv/sola
 or create an issue and assign it to yourself.
 
 ### Contribution Rewards
-First 25 succesful contributers to the Solana Cookbook will get an exculsive Glasseater NFT from the [Superteam DAO](https://superteam.fun). Use this NFT to get direct access to exclusive Solana Ecosystem jobs. If you have already contirbuted, [claim your Glasseater NFT now](https://alpha.layer3.xyz/task/contribute-to-the-solana-cookbook)!
+The first 25 successful contributors to the Solana Cookbook will get an exclusive Glasseater NFT from the [Superteam DAO](https://superteam.fun). Use this NFT to get direct access to exclusive Solana Ecosystem jobs. If you have already contributed, [claim your Glasseater NFT now](https://alpha.layer3.xyz/task/contribute-to-the-solana-cookbook)!
```
Comment (Agunderman10) [inline README.md:87]: As per @jacobcreech 's comment seen here: https://github.com/solana-dev-adv/solana-cookbook/pull/198#issuecomment-1013597730 we should probably add a note about requiring code changes to claim the NFT. We are also past 25 successful contributors so this note should be either removed or updated.

**Review by Agunderman10 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (jacobcreech) [reply README.md:87]: I thought this was added before. While we are past 25 contributors, I don't know if all have claimed their NFT

Comment (Agunderman10) [reply README.md:87]: Yea looks like, at the moment, the wording doesn't specify that the contributions need to be code changes. So looks like we just need to add that and this should be good. @AmaanSayyad

File: README.md [hunk 2]
```diff
@@ -28,7 +28,7 @@ Code Snippet
 
 ### Guides
 
-Guides are longer form informational documentation on various topics.
```
[the remark this answers is not in the document]
Comment (AmaanSayyad) [reply README.md:31]: Alright, thanks for the clarification.

File: README.md (hunk 1 above)
Comment (AmaanSayyad) [reply README.md:87]: Sure, will do.

**Comment by AmaanSayyad** [comment]
> Thanks for the smaller update. Just as a heads up, this PR would not be able to get the NFT if you're interested. It is only for code contributions.

Is Solana looking for Technical Writer? If yes do let me know. Thanks :)

**Comment by Agunderman10** [comment]
> > Thanks for the smaller update. Just as a heads up, this PR would not be able to get the NFT if you're interested. It is only for code contributions.
> 
> Is Solana looking for Technical Writer? If yes do let me know. Thanks :)

Looks like Neon labs and Metaplex are. Here are links for ya.
https://jobs.solana.com/companies/neon-labs/jobs/6127021-technical-writer#content
https://apply.workable.com/metaplex-studios/j/E3B94DD603/

**Review by Agunderman10 — APPROVED** [review]
lgtm

**Review by jacobcreech — APPROVED** [review]
Thanks for the update

**Pull request merged**